pyiron/lammps/structure.py

```diff
@@ -474,20 +474,31 @@ def structure_full(self):
             + " \n"
             + "{0:d} angles".format(len(angles_lst))
             + " \n"
-            + "{0} atom types".format(self._structure.get_number_of_species())
+            + "{0} atom types".format(len(sorted_species_list))
             + " \n"
             + "{0} bond types".format(num_bond_types)
             + " \n"
             + "{0} angle types".format(num_angle_types)
             + " \n"
         )
 
+        # atomtypes = list()
```

Comment:
If it is commented out, do we need it?

Reply:
No we don't. Fixed it here https://github.com/pyiron/pyiron/pull/457/commits/51714a0bd9098cb63844443ccfe19038f6c55e8b